library/core/src/mem/mod.rs

```diff
@@ -1254,12 +1254,10 @@ impl<T> SizedTypeProperties for T {}
 ///
 /// Nested field accesses may be used, but not array indexes.
 ///
-/// Enum variants may be traversed as if they were fields. Variants themselves do
+/// If the nightly-only feature `offset_of_enum` is enabled,
+/// variants may be traversed as if they were fields. Variants themselves do
 /// not have an offset.
```

Comment:
one last nit: while we're here, can we semantically linebreak this? https://sembr.org/
```suggestion
/// variants may be traversed as if they were fields.
/// Variants themselves do not have an offset.
```